src/oom.c

```diff
@@ -5,16 +5,14 @@
 #include <string.h>
 #include <unistd.h>
 
-#define OOM_SCORE "-1000"
-
-void attempt_oom_adjust()
+void attempt_oom_adjust(char* oom_score)
```

Comment:
nit can we do `const char* const` here?

Reply:
I think there may be a line change needed in `src/oom.h` as well

Reply:
done